src/components/capture/create/Spec.tsx

```diff
@@ -0,0 +1,64 @@
+import { Divider, Paper } from '@mui/material';
+import NewCaptureSpecForm from 'components/capture/create/SpecForm';
+import NewCaptureSpecFormHeader from 'components/capture/create/SpecFormHeader';
+import Error from 'components/shared/Error';
+import { Tables } from 'services/supabase';
+import { useQuery, useSelectSingle } from 'supabase-swr';
+
+interface ConnectorTag {
+    connectors: {
+        image_name: string;
+    };
+    id: string;
+    endpoint_spec_schema: string;
+    documentation_url: string;
+}
+
+interface Props {
+    connectorImage: string;
+    displayValidation: boolean;
+    readonly: boolean;
+}
+function NewCaptureSpec(props: Props) {
+    const { connectorImage, readonly, displayValidation } = props;
+    const tagsQuery = useQuery<ConnectorTag>(
+        Tables.CONNECTOR_TAGS,
+        {
+            columns: `
+                connectors(
+                    image_name
+                ),
+                id,
+                endpoint_spec_schema, 
+                documentation_url
+            `,
+            filter: (query) => query.eq('id', connectorImage),
+            count: 'exact',
+        },
+        [connectorImage]
+    );
+    const { data: connector, error } = useSelectSingle(tagsQuery, {});
```

Comment:
Parent wrapper for the spec form sections. This then passes down the needed details into the children.